app/scripts/image-transform.js

```diff
@@ -24,9 +24,45 @@ var imageTransform = (function(_) {
       }
       return images;
     },
+    image: function(data) {
+      var newData = {};
+      if(data && data.hits.hits.length > 0) {
+        newData = data.hits.hits[0]._source;
+
+        var adultService = {
+          total: newData.isImagePartOf.length,
+          array: []
+        };
+        newData.isImagePartOf.forEach(function(service) {
```

Comment:
I found an image where isImagePartOf was an object, not an array.  You should check if isImagePartOf is an array and, if not, convert it to one like this:
newData.isImagePartOf = _.isArray(newData.isImagePartOf) ? newData.isImagePartOf : [newData.isImagePartOf];